testsuites/syncgateway/functional/tests/conftest.py

```diff
@@ -156,8 +176,17 @@ def params_from_base_test_setup(request, params_from_base_suite_setup):
     cluster_topology = params_from_base_suite_setup["cluster_topology"]
     mode = params_from_base_suite_setup["mode"]
     xattrs_enabled = params_from_base_suite_setup["xattrs_enabled"]
+    sg_lb = params_from_base_suite_setup["sg_lb"]
 
     test_name = request.node.name
+
+    if sg_lb:
+        # These tests target one SG node
```

Comment:
online/offline can be made to work if we can figure out a way to broadcast from nginx, I couldn't find a way to do it. Log rotation tests run dd on the IP, so it creates a file on the load balancer and not on SG in this case. openidconnect uses the IP in the authenticate endpoints and toekns.